Vagrantfile

```diff
@@ -55,7 +55,7 @@ end
 
 # TODO(ksweeney): RAM requirements are not empirical and can probably be significantly lowered.
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
-  config.hostmanager.enabled = true
+  config.hostmanager.enabled = false
```

Comment:
@ewencp Good point about bringing up zk/broker cluster, that seems fixable. Note however that in this case, `vagrant up` by itself also won't work with zk cluster size > 1. This is noted in `vagrant/README.md` where they mention https://issues.apache.org/jira/browse/ZOOKEEPER-1506

I agree it's not ideal to ask for multiple steps to bring up a cluster, and that was the motivation for the wrapper script.

I actually tried the provisioner approach to avoid multiple steps on cluster bringup on local machines, but it doesn't quite work - it will only populate /etc/hosts with machines that are already up at the time of provisioning - so for example worker5 would have worker1->worker5 in its /etc/hosts, but none of the later workers.